src/tabs/AOCTab/SolutionView.jsx

```diff
@@ -1,24 +1,30 @@
-import Day1Solution from "./days/day1/Day1Script.js";
-import Day2Solution from "./days/day2/Day2Script.js";
-import Day3Solution from "./days/day3/Day3Script.js";
-import Day4Solution from "./days/day4/Day4Script.js";
+import React, { Suspense } from "react";
+import {ErrorBoundary} from 'react-error-boundary';
+// import { CodeBlock, ocean } from "react-code-blocks";
 
-const solutions = [];
-solutions[1] = <Day1Solution />
-solutions[2] = <Day2Solution />
-solutions[3] = <Day3Solution />
-solutions[4] = <Day4Solution />
+/*
+ *Dynamcally imports modules based on state
+ */
 
-// How do i get the day 1 component to show up on site?
+ function loadComponent(day) {
+  const Component = React.lazy(() =>
+    import("./days/day"+day+"/Day"+day+"Script.js")
+  );
+  return Component;
+}
 
-const SolutionView =(props) => {
+const SolutionView = (props) => {
   const day = props.parameter;
-  const answer = solutions[day]
+  const Component = loadComponent(day); 
   
   return(
-    <>
-      { answer }
-    </> 
+    <div>
+      <Suspense fallback={<div>Loading...</div>}>
+        <ErrorBoundary>

```

Comment:
React docs:   "Error boundaries are React components that catch JavaScript errors anywhere in their child component tree, log those errors, and display a fallback UI instead of the component tree that crashed"

I just put it there because the react.lazy( ) example I looked at was using it.  I'll delete it though.  When I comment them out, nothing changes